pkg/cmd/auth/setupgitcredential/setup.go

```diff
@@ -0,0 +1,46 @@
+package login
```

Comment:
This code looks like it doesn't belong in this PR. You can take it out and force-push to your branch.